### ISSUE TITLE
Crash bandicoot

### ISSUE THREAD

**Issue by Zierk59** [x360ce/x360ce#817]
Will be this game compatible?

**Comment by KiNGKiMO** [comment]
Yes, please use the latest ALPHA..
https://github.com/x360ce/x360ce/releases

**Comment by Zierk59** [comment]
Still I can't get it working :( Tried with Virtual device with no result

**Comment by KiNGKiMO** [comment]
You've the virtual device working at control panel (joy.cpl)?

![image](https://user-images.githubusercontent.com/11393200/42128602-835680a8-7caf-11e8-8acd-487151bbfc11.png)

**Comment by chrissdks** [comment]
yeah do the virtual device working at control thingi and it will work fine. This are the instructions https://github.com/x360ce/x360ce/blob/master/Wiki/BetaTesting.md

**Comment by Sequella** [comment]
Open de X360CE, and go to ''Game Settings'', click in ''Add'' and go to the folder of Crash Bandicoot, select de .exe, and mark the option 32-bit 1 and 4.
![sem titulo](https://user-images.githubusercontent.com/45779159/49777740-e8042c80-fce8-11e8-95ca-c58bf967fb4f.png)

**Comment by EvoJocys** [comment]
Sorry for the eight years late answer. The project was dormant from 2020 and has been active again since August 2026.

This report is about the 3.x line, where x360ce was an xinput DLL copied into each game's folder. The current 4.x line works differently: it creates a virtual Xbox 360 controller that every game sees, with nothing copied into game folders. Game-specific problems from the DLL era do not carry over.

The latest release is 4.21 (September 2026): https://github.com/x360ce/x360ce/releases

Closing as part of a tracker clean-up after the 4.21 release. If it still happens on 4.21, reopen this issue with the version and what you see.
